documentation/APIs/openapi3-core-advanced.yaml

```diff
@@ -0,0 +1,2607 @@
+
+  '/api/core/v3/congestion':
+    get:
+      tags:
+        - congestion
+      summary: Check readiness of the node to issue a new block, and the mana cost based on the rate setter and current network congestion.
+      description: >-
+        Check readiness of the node to issue a new block based on the rate setter and current network congestion. If no accountID is provided in the
+        query string parameter, return the congestion info of the node that the request is sent to.
+      parameters:
+        - in: query
+          name: accountID
+          schema:
+            type: string
+          example: "did:iota:123456789abcdefghi"
+          required: false
+          description: Identifier of the account to be checked for readines.
+        - in: query
+          name: slotIndex
+          schema:
+            type: integer
+          example: 1622568540
+          required: false
+          description: Check the cost and issuance readiness for a specific slot index.
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CongestionResponse'
+              examples:
+                CongestionResponse:
+                  $ref: '#/components/examples/get-congestion-estimate-response-example'
+        '400':
+          description: "Unsuccessful operation: indicates that the provided data is invalid."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestResponse'
+        '403':
+          description: "Unsuccessful operation: indicates that the endpoint is not available for public use."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ForbiddenResponse'
+        '500':
+          description: "Unsuccessful operation: indicates that an unexpected, internal server error happened which prevented the node from fulfilling the request."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalErrorResponse'
+        '503':
+          description: "Unsuccessful operation: indicates that the node is not synced."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceUnavailableResponse'
+
+  '/api/core/v3/outputs/{outputId}':
+    get:
+      tags:
+        - UTXO
+      summary: Find an output by its identifier.
+      description: Find an output by its identifier.
+      parameters:
+        - in: path
+          name: outputId
+          schema:
+            type: string
+          required: true
+          description: >-
+            Identifier of the output encoded in hex. An output is identified by
+            the concatenation of `transactionId+outputIndex`. Hex-encoded with 0x prefix.
```

Comment:
I think the concatenation here is not clear `outputIndex` (u16) needs to be converted to little endian first
there are multiple places where the same text is used

Reply:
@Thoralf-M  Thanks for reviewing!! All the fixes go to `openapi3-core.yaml`, the advanced ones are drafts for the next version, and it's not aligned to the `openapi3-core.yaml` yet.